backend/WebApi/Auth/JwtService.cs

```diff
@@ -115,7 +115,7 @@ public static LfUser ExtractLfUser(ClaimsPrincipal user)
         if (string.IsNullOrEmpty(emailClaim) || string.IsNullOrEmpty(idClaim) ||
             string.IsNullOrEmpty(roleClaim) || string.IsNullOrEmpty(projectRolesClaim))
         {
-            throw new ArgumentException($"User is missing required claims. Claims: {user.Claims}.");
+            return null;
```

Comment:
@hahn-kev I decided this makes more sense in the end, because when we get requests from unauthenticated users there can still be a WebContext, it just doesn't include a user.

Ultimately I made this change, because (to my surprise) my `ProjectAuthorizationHandler` gets instantiated for requests that don't need it. So, I either had to call `ExtractLfUser` from the authorization handler or make this change. I don't have a super strong opinion either way.

Reply:
My first thought was to just lazily get the user - and leave this code as is.

But that means we could miss bugs where we access the user from some code outside of an authorized context, in this case that code has to decide what to do if the user is null now.